docs/roadmap.md

```diff
@@ -65,14 +65,18 @@ If a roadmap item is a stub, **consider** opening a [GitHub issue](https://githu
 
 ### Migration
 
+The community is working on a number of automated scripts to help you migrate to Foam. The main work of developing such a method involves exporting your notes, converting them to the Markdown format, and then making sure that the links between notes (if you had those) still work.
+
 - [[migrating-from-roam]]
   - Discussion: [foam#55](https://github.com/foambubble/foam/issues/55)
 - [[migrating-from-obsidian]]
   - Discussion: [foam#46](https://github.com/foambubble/foam/issues/46)
-- [[Migrating from OneNote (stub)]]
+- [[Migrating from OneNote]]
```

Comment:
I believe this needs to match the filename, right @jevakallio ?
```suggestion
- [[migrating-from-onenote]]
```

Reply:
The first link works for me, so I'm guessing foam automatically converts link names to kebab-case before searching?

Reply:
Hmm, you're probably right, but I want to make sure with @jevakallio

Reply:
Yeah this needs to be changed to kebab-case. 

Also, you need to let Foam regenerate the link references at the bottom of the file. I believe this should happen in VS Code on save.

Reply:
Alright, I changed the kebab case, but for some reason the Foam -> Update Link References does nothing on the migrate-to-onenote.md file. I used it on roadmap.md and it worked fine, so I have no idea.